ngi_reports/utils/statusdb.py

```diff
@@ -65,38 +79,111 @@ def get_project_flowcell(self, project_id, open_date="2015-01-01", date_format="
+                        "db": self.db.name,
+                    }
         return project_flowcells
 
+
 class ProjectSummaryConnection(statusdb_connection):
     def __init__(self, dbname="projects"):
         super(ProjectSummaryConnection, self).__init__()
         self.db = self.connection[dbname]
-        self.name_view = {k.key:k.id for k in self.db.view("project/project_name", reduce=False)}
-        self.id_view = {k.key:k.id for k in self.db.view("project/project_id", reduce=False)}
+        self.name_view = {
+            k.key: k.id for k in self.db.view("project/project_name", reduce=False)
+        }
+        self.id_view = {
+            k.key: k.id for k in self.db.view("project/project_id", reduce=False)
+        }
+
 
 class SampleRunMetricsConnection(statusdb_connection):
     def __init__(self, dbname="samples"):
         super(SampleRunMetricsConnection, self).__init__()
         self.db = self.connection[dbname]
 
+
 class FlowcellRunMetricsConnection(statusdb_connection):
     def __init__(self, dbname="flowcells"):
         super(FlowcellRunMetricsConnection, self).__init__()
         self.db = self.connection[dbname]
-        self.name_view = {k.key:k.id for k in self.db.view("names/name", reduce=False)}
-        self.proj_list = {k.key:k.value for k in self.db.view("names/project_ids_list", reduce=False) if k.key}
+        self.name_view = {k.key: k.id for k in self.db.view("names/name", reduce=False)}
+        self.proj_list = {
+            k.key: k.value
+            for k in self.db.view("names/project_ids_list", reduce=False)
+            if k.key
+        }
+
 
 class X_FlowcellRunMetricsConnection(statusdb_connection):
     def __init__(self, dbname="x_flowcells"):
         super(X_FlowcellRunMetricsConnection, self).__init__()
         self.db = self.connection[dbname]
-        self.name_view = {k.key:k.id for k in self.db.view("names/name", reduce=False)}
-        self.proj_list = {k.key:k.value for k in self.db.view("names/project_ids_list", reduce=False) if k.key}
+        self.name_view = {k.key: k.id for k in self.db.view("names/name", reduce=False)}
+        self.proj_list = {
+            k.key: k.value
+            for k in self.db.view("names/project_ids_list", reduce=False)
+            if k.key
+        }
+
+
+class NanoporeRunConnection(statusdb_connection):
+    def __init__(self, dbname="nanopore_runs"):
+        super(NanoporeRunConnection, self).__init__()
+        self.db = self.connection[dbname]
+        self.name_view = {k.key: k.id for k in self.db.view("names/name", reduce=False)}
+        self.proj_list = {
+            k.key: k.value
+            for k in self.db.view("names/project_ids_list", reduce=False)
+            if k.key
+        }
+        # self.stats_view = {k.key:k.value for k in self.db.view("info/all_stats", reduce=False) if k.key}
+        # self.barcode_view = {k.key:k.value for k in self.db.view("info/barcodes", reduce=False) if k.key}
```

Comment:
Will these be used in the future?

Reply:
Hm, I don't remember 😅 I'll remove them for now.